kubernetes-archv3/README.md

```diff
@@ -0,0 +1,77 @@
+# Kubernetes Deployment Files
+
+This folder contains files to set up the CHT core App locally on Kubernetes. When run in Kubernetes, these files generate services that are needed to run the CHT core app.
+The services share secrets through environment variables in the  `secrets.yml` file. Services discover each other through these variables. If you update the names of any of the resources be sure to update the same in the `secrets.yml` file lest the services will not discover each other.
+Secrets in the `secrets.yml` file are base64 encoded. Be sure to encode them before adding them to the respective variable.
+To encode a secret you can run a command similar to the one below.
+
+`echo -n 'secret' | base64`
+
+To decode a secret you can run a command similar to the one below.
+
+`echo -n 'QWxhZGRpbjpvcGVuIHNlc2FtZQ==' | base64 --decode`
+
+You can deploy these files in you local Kubernetes set up from docker desktop on MACOS or using a Kubernetes engine like microk8s or k3s on Linux.
+
+## Steps to run this set up in a local Kubernetes set up. 
+
+1. Install a local Kubernetes engine like the one in  Docker desktop for MacOs or k3s for Linux steps shown below. 
```

Comment:
Following this step is confusing to me. The `k3s` install worked (pipe to `sudo sh -` ), but then I don't know which command to run next.  Do I also install `k3d`? The subsequent call to `k3d cluster` failed as I hadn't installed it. I suggest we remove commands that are not used.  I finally figured I can just skip them.